### PR TITLE
feat：translate article (Content, Tags, Abstract, Title) at article page.

### PR DESCRIPTION
**Pull request by Baihhh**

### CODE REVIEW

File: cmd/gopcomm/community_yap.gox [hunk 1]
```diff
@@ -361,50 +355,52 @@ post "/commit", ctx => {
...
+
 	id, _ = community.PutArticle(todo, uid, article)
 	ctx.json {
 		"code": 200,
 		"data": id,
 	}
-	// ctx.yap "edit", *article
+}
+
+get "/tranArticle", ctx => {
```
Comment (IRONICBo) [inline cmd/gopcomm/community_yap.gox:427]: Suggest to `/article/translation`

**Comment by Baihhh** [comment]
fix bug: #167 
image

**Comment by Baihhh** [comment]
fix bug: #170 
image

fix bug: #171 
image

**Comment by IRONICBo** [comment]
@Baihhh Please solve conflicts.

**Review by IRONICBo — APPROVED** [review]
LGTM

**Comment by xhyqaq** [comment]
Avoid magic values and use them as constants。 ："auto"

**Pull request merged**